### PR TITLE
Add a 401 page

### LINKED ISSUE #13: No 401 page

### PR DESCRIPTION
**Pull request by dfaulken**
From umts/jobapps.

Closes #13.

### CODE REVIEW

File: public/401.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Access Denied</title>
+  <!-- TODO: style-->
+</head>
+<body>
+  <div class="dialog">
+    <h1>You do not have permission to access this page.</h1>
+    <p>If you believe you should have access, please contact the site administrator.</p>
```
Comment (Anbranin) [inline public/401.html]: Add contact email, or is that not a thing that's done on a page like this?

Comment (dfaulken) [inline public/401.html]: Up to you.

Comment (Anbranin) [inline public/401.html]: include `transit-it@umass.edu`. Why not?

Comment (dfaulken) [inline public/401.html]: https://github.com/umts/st-pax/pull/18/commits/2a4dac973bf38635363e3c8115664fa4c3b0a308

**Pull request merged**